assets/blueprints/faucet/src/lib.rs

```diff
@@ -15,7 +16,7 @@ mod faucet {
         ) -> Global<Faucet> {
             Self {
                 vault: Vault::with_bucket(bucket),
-                transactions: KeyValueStore::new(),
+                transactions: KeyValueStore::new_with_registered(),
```

Comment:
Minor: not too big of a fan of the name, would like it to be either shorter (e.g. `new_bp_typed`) or more descriptive (`new_with_registered_type`). Middle of the road feels like worse of both worlds?

Reply:
Slightly worried about `_bp_`. Will go `new_with_registered_type`.